src/ceph-volume/ceph_volume/devices/raw/list.py

```diff
@@ -18,6 +19,28 @@ def direct_report(devices):
     _list = List([])
     return _list.generate(devices)
 
+def _get_bluestore_info(dev):
+    out, err, ret = process.call([
+        'ceph-bluestore-tool', 'show-label',
+        '--dev', dev], verbose_on_failure=False)
+    if ret:
+        # ceph-bluestore-tool returns failure if device is not bluestore OSD
+        logger.debug('failed to get BlueStore info on device %s' % dev)
```

Comment:
One more thing, let's print `out` and `err` as well, we return `None` in 2 places and we can't differentiate them. If we print `out` and `err` we know `ceph-bluestore-tool` failed to run.

Perhaps let's replace `ret` with `rc`?

Reply:
That makes sense. It is frustrating that c-b-t doesn't have a special return code or defined output identifier that lets us differentiate "is not a bluestore block" from other types of failures.